builder-base/install.sh

```diff
@@ -64,16 +73,15 @@ function build::go::install(){
 
     # AL2 provides a longer supported version of golang, use AL2 package when possible
     local yum_provided_versions="1.13"
-    local eks_built_versions="1.16.15 1.15.15"
+    local eks_built_versions="1.16.15 1.15.15 1.17.13 1.18.7 1.19.2"
     if [[ $eks_built_versions =~ (^|[[:space:]])${version}($|[[:space:]]) && $TARGETARCH == "amd64" && $IS_AL22 == false ]]; then
-        local artifacts_bucket='eks-d-postsubmit-artifacts'
         local arch='x86_64'
```

Comment:
We have arm right? Can we change this to support?

Reply:
updated. How's that look to you?